### ISSUE TITLE
Vendor App - Isolating Title or Library

### ISSUE THREAD

**Issue by kllittle** [CARLI/select#39]
I have found that when using the Quick Pricing tool in the Vendor App we are not able to isolate individual titles or libraries.  The Select All and Select None feature is working but the check boxes next to the titles/libraries is not working.

**Comment by mekane** [comment]
@kllittle is this universal? Can you please provide specific cycles and vendors where this happens?

**Comment by kllittle** [comment]
@mekane I have found the same to happen for title and library for EBSCO and the same for library for EBSCO.

**Comment by mekane** [comment]
@kllittle is this just when you masquerade, or have you noticed if it happens for regular vendor users as well?

**Comment by kllittle** [comment]
Hello, 
We are going to be having PQ in the system this week and they will need the ability to isolate titles when entering pricing.

**Comment by kllittle** [comment]
@mekane  This is something that happens when masquerading and when signing in as the vendor.  Sorry, I thought I had answered this question.

**Comment by mekane** [comment]
We found a problem with the checkboxes that was causing them to not work if the user managed to click on a specific part (the upper-left part of the checkbox). This was due to an additional fix made to allow clicking anywhere on the whole box to toggle the product or library. We made a quick fix for that, but we don't feel that the current solution is as good as it could be. And it's not 100% accessible at the moment.
